test/private/hasTestFailed.m

```diff
@@ -0,0 +1,13 @@
+function bool = hasTestFailed(status)
```

Comment:
Because this allows to call it from `countNumberOfErrors`, which is simplified quite a bit this way.